packages/edge/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "@logtail/edge",
+  "version": "0.4.6",
+  "description": "Logtail.com - Edge runtime logger",
```

```suggestion
  "description": "Better Stack Edge runtime logger (formerly Logtail)",
```

Let's please use Better Stack instead of Logtail. 🙏

Could you also add "Better Stack" and "Better Stack Logs" to keywords (keep Logtail)?

---

I'll unify that in a new PR, it's still Logtail everywhere, probably